lib/core/configurator.c

```diff
@@ -1259,7 +1259,8 @@ int h2o_configurator__do_parse_mapping(h2o_configurator_command_t *cmd, yoml_t *
             return -1;
         }
         if ((keys[j].type_mask & (1u << element->value->type)) == 0) {
-            char permitted_types[32] = "";
+            char permitted_types[40] = "";
+            PTLS_BUILD_ASSERT(sizeof(permitted_types) >= sizeof(" or a scalar or a sequence or a mapping"));
```

Comment:
Maybe change it to the following? There is no need to have a separate assert if we embed the condition that we want to check in the declaration.
```c
char permitted_types[sizeof(" or a scalar or a sequence or a mapping")] = "";
```

Reply:
Oh neat! Wondering why I did not come up with this 😄

Reply:
Done in 05af30f89a809f86ff921a5e30ce14a2a3ce7895